apps/old/widget/components.jsx

```diff
@@ -7,9 +7,6 @@ const { Step } = VM.require("${config_account}/widget/components.Step");
 const { InputField } = VM.require(
   "${config_account}/widget/components.InputField",
 );
-const { UploadField } = VM.require(
-  "${config_account}/widget/components.UploadField",
-);
```

Comment:
why are we removing it from here?

Reply:
Hi , As hooks are unavailable for modules and I have used this
<Widget
src="${alias_old}/widget/components.UploadField"
props={{
image: avatar,
onChange: (image) => setAvatar({ image }),
}}